go/chat/sender.go

```diff
@@ -107,6 +107,42 @@ func (s *BlockingSender) addPrevPointersToMessage(ctx context.Context, msg chat1
 	return updated, nil
 }
 
+func (s *BlockingSender) getAllDeletedEdits(ctx context.Context, msg chat1.MessagePlaintext,
+	convID chat1.ConversationID) (chat1.MessagePlaintext, error) {
+
+	// Make sure this is a valid delete message
+	if msg.ClientHeader.MessageType != chat1.MessageType_DELETE {
+		return msg, nil
+	}
+	if msg.ClientHeader.Supersedes == 0 {
+		return msg, fmt.Errorf("getAllDeletedEdits: no supersedes specified")
+	}
```

Comment:
I think it is ok. Either the caller has some bogus data in there, or a subset of the right set of messages IDs they had no power to create.